src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksPanel.qml

```diff
@@ -97,16 +97,8 @@ Item {
                     view.interactive = true
                 }
 
-                onDuplicateRequested: {
-                    tracksModel.duplicateTrack(model.index)
-                }
-
-                onDeleteRequested: {
-                    tracksModel.deleteTrack(model.index)
-                }
-
                 onOpenEffectsRequested: {
-                        tracksModel.openEffectsForTrack(model.index)
+                    tracksModel.openEffectForTrack(model.index)
```

Comment:
shouldn't it be Effects? (looking at Q_INVOKABLE function in trackslistmodel)
```qml
tracksModel.openEffectsForTrack(model.index)
```

Reply:
oops, typo, thanks